README.md

```diff
@@ -64,53 +64,12 @@ The Open Charge Point Protocol (OCPP) is a network protocol for communication be
 
 ## Usage
 
-There is an example OCPP 1.5-JSON client application in ocpp-json/src/test/scala/com/thenewmotion/example, reproduced
-here for your convenience:
+There is an example OCPP 1.5-JSON client application in example-json-client/src/main/scala/com/thenewmotion/ocpp/json. You can run it like this:
 
-```scala
-package com.thenewmotion.example
+    sbt "project example-json-client" "run 01234567 ws://localhost:8017/ocppws abcdef1234abcdef1234abcdef1234abcdef1234"
```

Comment:
I think here is also interesting to put an explanation of what are the parameters sent, what does the password should look like? what is the format? what is the 012...?